tests/e2e/cases/tm2eth/scripts/handshake

```diff
@@ -2,6 +2,12 @@
 
 set -eux
 
+if [ "x$USE_FAKELOST" = "xyes" ]; then
+    export DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_ibc1="10"
+    export DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER_ibc1="10"
```

Comment:
@dai1975 These names of the environment variables are wrong and ignored by the relayer.